README.md

```diff
@@ -106,6 +132,38 @@ However people have been using custom rich presence for past 4-5 years and there
 ## Translate
 See [Contribute for Translation](https://github.com/dead8309/Kizzy/issues/2#issue-1370365856)
 
+## Contributing
```

Comment:
I'll add the `Contributing.md` file later. No need for this as of now

Reply:
Got it, thanks.